src/consts.js

```diff
@@ -1,216 +0,0 @@
-'use strict';
```

Comment:
Well, this file is totally replaced in a different way. 
Also, this is git's affordance problem.     
According to Don Norman, never blame user's fault if the interface sucks.